portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,39 +25,47 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
   
-  private ArrayList<String> quotes;
-
-  public void init() {
-    quotes = new ArrayList<>();
-    quotes.add("Welcome");
-    quotes.add("Bienvenidos");
-    quotes.add("Bonjour");
-    quotes.add("Aloha");
-  }
+  private List<String> entries = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    String json = convertToJson(quotes);
+    String json = convertToJson(entries);
     response.getWriter().println(json);
   }
 
-  private String convertToJson(ArrayList<String> quotes) {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String name = getParameter(request, "name", "");
+    String email = getParameter(request, "email", "");
+    String comment = getParameter(request, "comment", "");
+    entries.add(name);
```

Comment:
You can inline the gets. Since you don't use the vars for anything other than using in the add, you can just directly call
entries.add(getParameter(request, "name", "");
it makes the code more compact

Reply:
I'm not exactly sure what you mean by incline. Let me know if the change is correct.

Reply:
Wow I read that wrong. Ignore that last comment. I understand now. lol

Reply:
Done